From: Borislav Petkov <petkovbb@googlemail.com>
Subject: x86, cpu: mv display_cacheinfo -> cpu_detect_cache_sizes
Patch-mainline: 2.6.33-rc1
References: bnc#564618, FATE#306952
Git-commit: 27c13ecec4d8856687b50b959e1146845b478f95

display_cacheinfo() doesn't display anything anymore and it is used to
detect CPU cache sizes. Rename it accordingly.

Signed-off-by: Borislav Petkov <petkovbb@gmail.com>
LKML-Reference: <20091121130145.GA31357@liondog.tnic>
Signed-off-by: H. Peter Anvin <hpa@zytor.com>
Signed-off-by: Rafael J. Wysocki <rjw@suse.de>
Automatically created from "patches.arch/x86-cpu-mv-display_cacheinfo-cpu_detect_cache_sizes.patch" by xen-port-patches.py

--- head-2010-01-18.orig/arch/x86/kernel/cpu/common-xen.c	2010-01-18 17:05:18.000000000 +0100
+++ head-2010-01-18/arch/x86/kernel/cpu/common-xen.c	2010-01-18 17:05:23.000000000 +0100
@@ -69,7 +69,7 @@ void __init setup_cpu_local_masks(void)
 static void __cpuinit default_init(struct cpuinfo_x86 *c)
 {
 #ifdef CONFIG_X86_64
-	display_cacheinfo(c);
+	cpu_detect_cache_sizes(c);
 #else
 	/* Not much we can do here... */
 	/* Check if at least it has cpuid */
@@ -411,7 +411,7 @@ static void __cpuinit get_model_name(str
 	}
 }
 
-void __cpuinit display_cacheinfo(struct cpuinfo_x86 *c)
+void __cpuinit cpu_detect_cache_sizes(struct cpuinfo_x86 *c)
 {
 	unsigned int n, dummy, ebx, ecx, edx, l2size;
 
